commands/eject.js

```diff
@@ -216,7 +216,7 @@ function configurePackage(bare) {
 	app.eslintConfig = eslintConfig;
 	app.eslintIgnore = app.eslintIgnore || [];
 	app.eslintIgnore = app.eslintIgnore.concat(eslintIgnore.filter(l => !app.eslintIgnore.includes(l)));
-	backupOld(['.eslintignore', '.eslintrc.js', '.eslintrc.yaml', '.eslintrc.yml', '.eslintrc.json', '.eslintrc']);
+	backupOld(['.eslintignore', 'eslint.config.js', '.eslintrc.yaml', '.eslintrc.yml', '.eslintrc.json', '.eslintrc']);
```

Comment:
i'm not sure what to do about the other file formats. I'll need someone else's opinion

Reply:
Yes, I would appreciate someone else's opinion as well.

Reply:
backupOld() is the function that if parameters(file names) exist in the project, saves the backup file by attaching *.old.
I recommend keeping .eslintrc.js and adding eslint.config.js to the parameter.